deploy/role.yaml

```diff
@@ -31,6 +32,12 @@ rules:
   - statefulsets
   verbs:
   - '*'
+- apiGroups:
```

Comment:
I think `pods/log` should be put in the `apiGroups` just above (with `pods` and `pods/exec`)